language/diem-framework/modules/DiemConfig.move

```diff
@@ -420,6 +414,8 @@ module DiemConfig {
                 config_address == CoreAddresses::DIEM_ROOT_ADDRESS();
 
         /// After genesis, no new configurations are added.
+        /// TODO(mengxu): this invariant looks suspicious to me. I don't see how this can hold in the
+        /// `publish_new_config_and_get_capability` function (which is a public function)
         invariant update
             DiemTimestamp::is_operating() ==>
                 (forall config_type: type where spec_is_published<config_type>(): old(spec_is_published<config_type>()));
```

Comment:
This is the only global invariant that cannot verify (and it shouldn't) @wrwg @DavidLDill 

After this PR, the prover shows that this invariant does not hold in the `DiemConfig::publish_new_config_and_get_capability` function (which is a public function). And the prover is right, there is nothing in the function that stops DiemRoot from publishing a new config by calling this function.